Model/ResourceModificationTimeRepository.php

```diff
@@ -13,20 +13,21 @@ public function __construct(Context $dbContext, $connectionName = null)
         $this->dbConnection = $dbContext->getResources()->getConnection($connectionName);
     }
 
-    public function getLastModificationTime(string $identifier)
+    public function getLastModification(string $identifier)
```

Comment:
I agree. `getMessageGroup` is better. I am going to make all related names consistent.